### PR TITLE
Remove references to LMA, as we will use a topic page for COS

### PR DESCRIPTION
**Pull request by mmanciop**
## Done
- Remove references to LMA, as we will use a topic page for COS

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card
Fixes #

## Screenshots
[if relevant, include a screenshot]

### CODE REVIEW

**Comment by Lukewh** [comment]
Could you resolve the conflict please @mmanciop? Once that's done we're waiting on confirmation that the topics pages are ready to go live – once live, we'll land this too, thanks @mmanciop :)

**Review by jkfran — APPROVED** [review]
LGTM

**Pull request merged**
